lib/iris/fileformats/grib/grib_phenom_translation.py

```diff
@@ -190,10 +190,10 @@ def _make_grib2_cf_entry(param_discipline, param_category, param_number,
                               '(skipping).'.format(standard_name))
                 return None
         # convert units string to iris Unit (i.e. mainly, check it is good)
-        iris_units = iris.unit.Unit(units)
+        cf_unitss = cf_units.Unit(units)
```

Comment:
these aren't wrong, per se

an instance of a Unit is being created, this was being called iris_units before; now it is being called cf_unitss to give a a unique name.

however,  I agree it reads oddly, so I'll change them where they occur